### PR TITLE
Add support for axe auditing with safari

### PR DESCRIPTION
**Pull request by rousik**
This code adds `--axe` argument that allows fully automated audit of pages using axe. 

- `requestedUrl` is added to the audit results to identify the page that generated these audits.
- `_AXE_` is used when constructing the audit filenames.
- `driver.quit()` ensures that the browser is properly torn down. close() sometimes leaves garbage behind and interferes with the next iteration.
- `jira_ticket` positional argument replaced with `--ticket $blah`.

For `auto-lighthouse` I have used `--ticket N` and `--output-path` arguments that combined to `${output_path}/C4G-${ticket}_CD_${index}.json`. Here the --ticket seems to expect to have form of `C4G-${N}`. That's okay but we might consider consolidating these two.

### CODE REVIEW

**Comment by ems831** [comment]
Looks good to me.  I ran it against one of the batches I did manually.  Big improvement!

File: public/tools/auto-audit/auto-audit.py [hunk 1]
```diff
@@ -16,10 +17,14 @@
 
 parser = argparse.ArgumentParser(description='Automatically Process the URLs from a given XLSX file.')
 parser.add_argument('--start-at', type=int, default=2)
-parser.add_argument('jira_ticket')
+parser.add_argument('--axe', default=False, action='store_true')
```
Comment (Winerip) [inline public/tools/auto-audit/auto-audit.py:20]: nit: This is not naturally a boolean, and becomes very un-intuitive to use if the default were to change, as was discussed. To run the webInspector version, you'd need --axe=false. Perhaps a String (I'm thinking Java flags, I'd use an enum, not sure that makes sense in python though).

Comment (rousik) [reply public/tools/auto-audit/auto-audit.py:20]: I have switched this to `--axe', '--no-axe` flag pair which should allow the bool-like behavior.

Comment (rousik) [reply public/tools/auto-audit/auto-audit.py]: After off-line discussion I have switched this to `--mode={axe,manual}` flag and fixed one stale reference to `args.jira_ticket`

**Pull request merged**